### PR TITLE
Move prettier config file to root directory

### PR DESCRIPTION
**Pull request by khanguslee**
## Description

Did a clean clone of the repository. VS code was incorrectly formatting the js file. By moving the `.prettierrc` file that was already in the `/server` directory, into the root directory, I was able to get vs code to autoformat the JS code using the prettier formatter.

## Screenshots

N/A

## Steps to Test

Make some changes in a file, save file whilst in vs code. VS code should format the document automatically.

### CODE REVIEW

**Comment by khanguslee** [comment]
Seems like the code that was formatted in the repo does not represent the eslint and prettier rules we have set in the repo. We will need to discuss what we want and don't want asap.

**Review by emilytrau — CHANGES_REQUESTED** [review]
@khanguslee ouch my review stats!!!

**Comment by khanguslee** [comment]
> @khanguslee ouch my review stats!!!

If you set WIP label it doesnt count pree sure

**Review by hallgchris — CHANGES_REQUESTED** [review]
Just reviewing so Pull Reminders shuts up haha

**Comment by hallgchris** [comment]
@khanguslee What's the status on this?

**Comment by khanguslee** [comment]
Probably a few controversial changes in here, btw. But at least we can all conform to a single style and code format.

Didn't have to change any of the existing eslint rules! Most of the changes in the files, is prettier just putting stuff in a new line etc.

LMK if you have any concerns or not. I also encourage everyone to have prettier to format your files upon saving.

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -24,8 +25,16 @@ jobs:
...
           cd client
           yarn
           yarn lint
+          cd ..
+
+      - name: Run linter on server
+        run: |
+          cd server
+          yarn
+          yarn eslint-check
```
Comment (hallgchris) [inline .github/workflows/linter.yml]: A little weird that we have different commands for client/server

**Review by hallgchris — DISMISSED** [review]
Looks good to me! All things considered, our repo was quite neat already

**Comment by khanguslee** [comment]
> Looks good to me! All things considered, our repo was quite neat already

Yeah it already was! As we get more devs, I thought it would prevent us from reviewing code format, and start to review the contents of the code instead.

File: server/package.json [hunk 2]
```diff
@@ -5,8 +5,8 @@
   "main": "server/server.js",
   "scripts": {
     "eslint": "eslint",
-    "eslint-check": "eslint --print-config . | eslint-config-prettier-check",
-    "eslint-fix": "eslint --fix",
+    "lint": "eslint --print-config . | eslint-config-prettier-check",
+    "lint:fix": "eslint --fix",
```
Comment (hallgchris) [inline server/package.json:9]: Can we have lint & fix on the client too?

**Review by hallgchris — APPROVED** [review]
Looks good!

**Comment by khanguslee** [comment]
Just need your review now @angustrau. We can finally close this PR up!

**Review by emilytrau — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**